src/domain/application/AppConfig.ts

```diff
@@ -0,0 +1,94 @@
+class AppConfig {
+  static get apiUrl() {
+    return getEnvOrError(process.env.REACT_APP_API_URI, 'REACT_APP_API_URI');
+  }
+
+  static get oidcAuthority() {
+    const origin = getEnvOrError(
+      process.env.REACT_APP_OIDC_AUTHORITY,
+      'REACT_APP_OIDC_AUTHORITY'
+    );
+    return new URL(origin).href;
+  }
+
+  /**
+   * The audiences used in the OIDC.
+   * 
+   * @example
+   * // In Tunnistamo it can be left undefined.
+   * ["https://api.hel.fi/auth/kukkuu"]
+   * // In Keycloak:
+   * [
+        'kukkuu-api-test',
+        'profile-api-test',
+      ]
+   */
+  static get oidcAudience() {
+    return process.env.REACT_APP_OIDC_AUDIENCES;
+  }
+
+  static get oidcClientId() {
+    return getEnvOrError(
+      process.env.REACT_APP_OIDC_CLIENT_ID,
+      'REACT_APP_OIDC_CLIENT_ID'
+    );
+  }
+
+  static get oidcScope() {
+    return getEnvOrError(
+      process.env.REACT_APP_OIDC_SCOPE,
+      'REACT_APP_OIDC_SCOPE,'
+    );
+  }
+
+  static get oidcReturnType() {
+    // "code" for authorization code flow.
+    return process.env.REACT_APP_OIDC_RETURN_TYPE ?? 'code';
+  }
+
+  static get oidcKukkuuApiClientId() {
+    return (
+      process.env.REACT_APP_OIDC_KUKKUU_API_CLIENT_ID ?? this.oidcKukkuuAPIScope
+    );
+  }
+
+  static get oidcKukkuuApiTokensUrl() {
+    return this.oidcServerType === 'KEYCLOAK'
+      ? `${this.oidcAuthority}protocol/openid-connect/token`
+      : `${this.oidcAuthority}api-tokens/`;
+  }
+
+  static get oidcKukkuuAPIScope() {
+    return getEnvOrError(
+      process.env.REACT_APP_KUKKUU_API_OIDC_SCOPE,
+      'REACT_APP_KUKKUU_API_OIDC_SCOPE'
+    );
+  }
```

Comment:
yes, I did it in purpose and most probably they usually are the same (for Kukkuu Admin UI). I did it like this to avoid changes needed to be done to the pipeline configuration.